devtest-unit-test-java/src/test/resources/v3/rrpair/operation-9-req.txt

```diff
@@ -0,0 +1,6 @@
+GET /import/test/operation-9 HTTP/1.1
+Content-Type: application/json
+X-Api-Key: 89ba4c3d-3373-40ad-8a46-aec33f944fae

```

Comment:
:exclamation:**Cycode**: Secret of type: 'Generic Password' was found.
Severity: High

Would you like to exclude this secret from your status checks?
Tell us what to do with one of the following hashtags:

| Tag | Short Description |
| --- | --- |
| **#cycode_secret_false_positive** | Applies to this secret value for all repos in your organization |
| **#cycode_secret_ignore_everywhere** | Applies to this secret value for all repos in your organization |
| **#cycode_secret_ignore_here** | Applies to this request only |
| **#cycode_secret_revoked** | Applies to this secret value for all repos in your organization |

 <!-- 6e1f03fe-50df-4b51-aa80-64efc16e4111 -->